src/coreclr/scripts/superpmi.py

```diff
@@ -185,7 +176,7 @@
 
 merge_mch_pattern_help = """\
 A pattern to describing files to merge, passed through directly to `mcs -merge`.
-Acceptable patterns include `*.mch`, `file*.mch`, and `c:\my\directory\*.mch`.
+Acceptable patterns include `*.mch`, `file*.mch`, and `c:\\my\\directory\\*.mch`.
```

Comment:
Probably worth mentioning that it does recursive search inside `c:\\my\\directory` with the pattern as it was not obvious when I first saw the help message.